src/foam/java/Class.js

```diff
@@ -47,8 +47,20 @@ foam.CLASS({
       name: 'static',
       value: false
     },
-    'abstract',
-    'extends',
+    {
+      class: 'Boolean',
+      name: 'final',
+      value: false
```

Comment:
Default value for value: is false so this line is redundant here and for abstract.